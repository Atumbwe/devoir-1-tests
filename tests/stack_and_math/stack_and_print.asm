push 300
push 200
stack
print
