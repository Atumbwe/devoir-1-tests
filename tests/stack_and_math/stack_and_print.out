[ 300 200 ]
200
